consensus/safety-rules/src/safety_rules.rs

```diff
@@ -403,14 +403,9 @@ impl SafetyRules {
         }
 
         self.verify_qc(block_data.quorum_cert())?;
-        if self.verify_and_update_preferred_round(block_data.quorum_cert(), &mut safety_data)? {
-            self.persistent_storage.set_safety_data(safety_data)?;
-        }
```

Comment:
oh I see, you have to read this carefully

Reply:
I guess I would actually argue that signing anything isn't really that great. I'm happy to agree with @JoshLind who will most certainly take my side, maybe :P ... though recent track records suggest we are in for a healthy debate.

Reply:
I don't disagree that signing anything isn't great (otherwise I could just pass a hash value here), I don't really mind checking the preferred round either if that makes you feel better as long as we don't persist it